app/views/loans/index.html.erb

```diff
@@ -1,43 +1,159 @@
-<p id="notice"><%= notice %></p>
-
-<h1>Loans</h1>
-
-<table>
-  <thead>
-    <tr>
-      <th>Current balance</th>
-      <th>Original amount</th>
-      <th>Interest rate</th>
-      <th>Periods in year</th>
-      <th>User</th>
-      <th>Loan name</th>
-      <th>Monthly min payment</th>
-      <th>Schedule hash</th>
-      <th>Adj schedule hash</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @loans.each do |loan| %>
+<div class="row mb-3">
+  <div class="col-md-12">
+    <h1>
+      All loans
+    </h1>
+
+    <a href="/loans/new" class="btn btn-block btn-outline-secondary">
+      Add a new loan
+    </a>
```

Comment:
You could use a `link_to` here and route helper.

Reply:
Updated.  Thanks!